source/tyr/tyr/tasks.py

```diff
@@ -169,6 +169,9 @@ def import_data(
             current_app.logger.info("skipping mimir import")
 
         actions.append(finish_job.si(job.id))
+
+        # We should delete old backup directories related to this instance
+        actions.append(purge_instance.si(instance.id, current_app.config['DATASET_MAX_BACKUPS_TO_KEEP']))
```

Comment:
this is going to greatly improve our resilience :+1:

Reply:
Do you think the CRON job is still needed ? Should we remove it at the same time ?

Reply:
We can leave for some time and remove it after a verification.